client/my-sites/sidebar/sidebar.jsx

```diff
@@ -371,7 +382,10 @@ module.exports = React.createClass( {
 	},
 
 	trackUpgradeClick: function() {
-		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', { cta_name: 'sidebar_upgrade_default' } );
+		analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', {
+			cta_name: 'sidebar_upgrade_default',
```

Comment:
It probably isn't the best name for our test, but `sidebar_upgrade_default` was being used here before any of our code. We might want to keep this consistent in case any existing reports are relying on this. I think it's appropriate to just add a `cta_landing` attribute, since that's all that we're changing.